src/main/java/jp/co/soramitsu/sora/crypto/algorithms/SignatureSuiteRegistry.java

```diff
@@ -0,0 +1,45 @@
+package jp.co.soramitsu.sora.crypto.algorithms;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SignatureSuiteRegistry {
```

Comment:
Having static-methods-only class is considered as a code smell, especially if it manages state (while classes which methods are stateless are permitted, such as Utility classes)

Such classes automatically are harder to mock (developer will have to find alternatives to Mockito)

Harder to extend its' logic

As an alternative to static methods I suggest using singleton (but this also implies class is aware of multithreaded environment)

Reply:
@tdorzhi it is supposed to be changed at compile time, so you register your crypto suites once and never change. I don't think it should be aware about concurrency, since we will not have concurrent writes.